<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#444444;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>542</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #444444;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="insertBut">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>260</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="removespin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindMinBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Find Min</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindMaxBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Find Max</string>
    </property>
   </widget>
   <widget class="QLabel" name="findMinLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>430</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 	background-color:#333333;
	color: white;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="findMaxLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 	background-color:#333333;
	color: white;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BalanceBut">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QLabel" name="findLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 	background-color:#333333;
	color: white;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="findSpin">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FindAverageBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Find Average</string>
    </property>
   </widget>
   <widget class="QLabel" name="findAverage">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> 	background-color:#333333;
	color: white;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="preOrderEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>40</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printPreOrderBut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Print PreOrder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printInOrderBut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Print InOrder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="printPostOrderBut">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Print PostOrder</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inOrderEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="postOrderEdit">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearBut">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #333333;
    color: #ffffff;
    font-family: &quot;Arial Bold&quot;;
    font-size: 20px;
    border-radius: 10px;
    
}

QPushButton:hover {
    background-color: #444444;
}

QPushButton:pressed {
    background-color: #222222;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
